types/black-engine/index.d.ts

```diff
@@ -0,0 +1,168 @@
+// Type definitions for black-engine 0.5
+// Project: https://github.com/MassiveHeights/Black (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
Please remove instructional comments provided by template

Reply:
Removed in this commit https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59390/commits/37ddde8e584ba1ebd9ceb124712eec4af27969c6